geomesa-blobstore/geomesa-blobstore-core/src/test/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStoreTest.scala

```diff
@@ -38,10 +38,29 @@ class AccumuloBlobStoreTest extends Specification {
 
   val testfile1 = "testFile.txt"
   val testfile2 = "testFile2.txt"
+  val testfile3 = "testFile3.txt"
   var testFile1Id = ""
 
   "AccumuloBlobStore" should {
-    "be able able to store and retrieve a file" in {
+
+    "be able to store and delete a file" in pending("Deleting from accumulodatastore may not be working in blobstore") {
```

Comment:
hit this with the debug:

https://github.com/ccri/geomesa/blob/fcr_blobstore_rest_improvements/geomesa-blobstore/geomesa-blobstore-core/src/main/scala/org/locationtech/geomesa/blob/core/AccumuloBlobStore.scala#L51

call ```sf.getFeatureID```